cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -470,37 +470,24 @@ def _verify_deprecated_info(self, course_id, advanced_modules, info, deprecated_
         )
 
     @ddt.data(
-        {'publish': True},
-        {'publish': False},
+        [{'publish': True}, ['notes']],
+        [{'publish': False}, ['notes']],
+        [{'publish': True}, ['notes', 'lti']]
```

Comment:
Note to reviewers: I combined two test cases to remove code duplication.

Also note that it wasn't actually necessary to specify DEPRECATED_BLOCK_TYPES because the helper method is being called directly (and the helper method is called with the list of deprecated block_types).